src/repository/mysqlRepository/walletObjectMappingMySQL.ts

```diff
@@ -28,4 +26,11 @@ export const mySqlData: DbObjectMapping = {
     });
```

Comment:
is the where empty ?  -- github cuts it of right on the line just after the where